src/hooks/useSubmitPersonalInfo.js

```diff
@@ -27,7 +27,7 @@ export const useSubmitPersonalInfo = () => {
           onSuccessNavigate(data.CV);
         }
       } else {
-        setError(data.error || 'Something went wrong.');
+        setError(data.errors[0] || 'Something went wrong.');
```

Comment:
for now i think would also be good to log these errors to the console so that we can debug it from the frontend as well!

eg.
```jsx
console.warn(data)
```